src/include/Makefile.am

```diff
@@ -126,8 +126,56 @@ noinst_HEADERS = \
 	site_sched_attr_enum.h \
 	site_svr_attr_def.h \
 	site_svr_attr_enum.h \
+	job_attr_enum.h \
+	node_attr_enum.h \
+	queue_attr_enum.h \
+	resc_def_enum.h \
+	resv_attr_enum.h \
+	sched_attr_enum.h \
+	svr_attr_enum.h \
 	svrfunc.h \
 	ticket.h \
 	tracking.h \
 	user.h \
 	work_task.h
+	win.h \
+	win_remote_shell.h \
```

Comment:
Aren't these going away?

Reply:
yes somehow happened after the rebase I guess, fixed